source/postprocess/memory_statistics.cc

```diff
@@ -44,17 +44,60 @@ namespace aspect
       statistics.add_value ("current_constraints memory consumption (MB) ", this->get_current_constraints().memory_consumption()/mb);
       statistics.add_value ("Solution vector memory consumption (MB) ", this->get_solution().memory_consumption()/mb);
 
+      if (output_vmpeak)
+        {
+          // allow disabling of the output because this is not stable in automated tests:
+          dealii::Utilities::System::MemoryStats stats;
+          dealii::Utilities::System::get_memory_stats(stats);
+          statistics.add_value ("Peak virtual memory usage (VmPeak) (MB) ", stats.VmPeak/1024.0);
+        }
+
       std::ostringstream output;
       output << std::fixed << std::setprecision(2) << this->get_system_matrix().memory_consumption()/mb << " MB";
 
       return std::pair<std::string, std::string> ("System matrix memory consumption: ",
                                                   output.str());
 
     }
+
+
+
+
+    template <int dim>
+    void
+    MemoryStatistics<dim>::declare_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection("Postprocess");
+      {
+        prm.enter_subsection("Memory statistics");
+        {
+          prm.declare_entry ("output virtual memory peak", "true",
```

Comment:
Close -- we usually start parameter names with upper case letters.

Reply:
Matter of fact, I think the proper name should be "Output peak virtual memory". Or maybe "Output peak virtual memory (VmPeak)".